build.gradle

```diff
@@ -4,19 +4,12 @@ plugins {
     id 'com.jfrog.bintray' version '1.8.4'
     id 'net.researchgate.release' version '2.7.0'
     id 'eclipse'
-    id 'org.testeditor.gradle-plugin' version '0.8'
 }
 
 description = """Testing framework for web applications using Selenium for automating browsers see https://www.seleniumhq.org/"""
 
 apply from: 'code-style/codestyle.gradle'
 
-// Configure the testeditor plugin
```

Comment:
Please add for aml checks during gradle build, but with actual version of xtext  ;O)